site/docs/tutorials/pulsar.md

```diff
@@ -133,7 +133,7 @@ shadowJar {
         <dependency>
             <groupId>com.hazelcast.jet.contrib</groupId>
             <artifactId>pulsar</artifactId>
-            <version>0.1-SNAPSHOT</version>
+            <version>0.2-SNAPSHOT</version>
```

Comment:
This should be aligned also with gradle file. Please change in gradle file `compile 'com.hazelcast.jet-contrib:pulsar:0.1-SNAPSHOT'` to `compile 'com.hazelcast.jet-contrib:pulsar:0.2-SNAPSHOT'`

Reply:
I reverted both to 0.1-SNAPSHOT, since it will be the first release.